stdlib/LibGit2/test/libgit2.jl

```diff
@@ -2424,7 +2525,10 @@ mktempdir() do dir
             @test err == exhausted_error
             @test auth_attempts == 3
             @test p.explicit == invalid_cred
-            @test p.credential != invalid_cred
+            # @test p.credential != invalid_cred # TODO!!!
```

Comment:
I'll try to look into this

Reply:
These tests can be re-enabled if you set the non-secret fields of [`UserPasswordCredential`](https://github.com/JuliaLang/julia/pull/27565/files#diff-32a4d2f02c3fb1e702e6f9cba528a2cdR1214) and [`SSHCredential`](https://github.com/JuliaLang/julia/pull/27565/files#diff-32a4d2f02c3fb1e702e6f9cba528a2cdR1251) to `""` in their respective `shred!` methods.